AllReadyApp/Web-App/AllReady/wwwroot/js/activityDetails.js

```diff
@@ -1,5 +1,5 @@
-﻿var HTBox;
-(function (HTBox) {
+var HTBox;
+(function (HTBox, $) {
```

Comment:
I don't like passing in a variable called `$` on jQuery projects that has a pretty specific meaning. Perhaps `container` instead.